dump25/sitelinks/text.py

```
@@ -0,0 +1,335 @@
+    "wikiquote": "{{int:Wikibase-otherprojects-wikiquote}}",
+    "wiktionary": "{{int:Wikibase-otherprojects-wiktionary}}",
+    "wikibooks": "{{int:Wikibase-otherprojects-wikibooks}}",
+    "wikinews": "{{int:Wikibase-otherprojects-wikinews}}",
+    "wikiversity": "{{int:Wikibase-otherprojects-wikiversity}}",
+    "wikivoyage": "{{int:Wikibase-otherprojects-wikivoyage}}",
+}
+
+
+def make_name(code):
+    # ---
+    names = {
```

Ruff (0.8.2)

Local variable `names` is assigned to but never used

Remove assignment to unused variable `names`

(F841)